icubam/db/synchronizer.py

```diff
@@ -186,13 +190,19 @@ def sync_users_from_csv(self, csv_contents: TextIO, force_update=False):
     self.sync_users(users_df, force_update)
     return users_df.shape[0]
 
-  def sync_bedcounts_from_csv(self, csv_contents: TextIO, force_update=False):
+  def sync_bedcounts_from_csv(
+    self, csv_contents: TextIO, force_update=False
+  ) -> pd.DataFrame:
     """Check that columns correspond, insert into a DF and synchronize."""
-    bedcounts_df = pd.read_csv(csv_contents)
+    bedcounts_df = pd.read_csv(
+      csv_contents,
+      parse_dates=BC_DATES_COLUMNS,
+      date_parser=lambda x: pd.to_datetime(x, utc=True)
```

Comment:
With something like
``` d["create_date"] = d.datetime.dt.tz_localize("Europe/Paris").dt.tz_convert(tz=tz.tzutc())``` 
you should be able load the time with the correct locale and then convert to UTC. This check will be compatible with the previous validation method in Synchronizer as well.

Reply:
Sorry, what's the difference with what I had previously?

Reply:
I went back to the previous solution and added a timezone parameter to avoid hardcoding it.